src/Laravel/routes/api.php

```diff
@@ -24,8 +24,8 @@
 use ApiPlatform\Metadata\Resource\Factory\ResourceNameCollectionFactoryInterface;
 use Illuminate\Support\Facades\Route;
 
-$globalMiddlewares = config()->get('api-platform.routes.middleware', []);
-$domain = config()->get('api-platform.routes.domain');
+$globalMiddlewares = config()->get('api-platform.routes.middleware') ?? [];
```

Comment:
```suggestion
$globalMiddlewares = config()->get('api-platform.routes.middleware') ?? [];
```

what was wrong using the default argument?
```suggestion
$globalMiddlewares = config()->get('api-platform.routes.middleware', []);
```